sbc_translation/wizards/reply_to_comments_view.xml

```diff
@@ -5,17 +5,9 @@
         <field name="model">sbc.reply.to.comments.wizard</field>
         <field name="arch" type="xml">
             <form string="Reply to comments">
-                <group>
-                    <field name="paragraph_ids">
-                        <tree default_order="page_id asc, sequence asc">
-                            <field name="original_text"/>
-                            <field name="translated_text"/>
-                            <field name="comments"/>
-                        </tree>
-                    </field>
-                </group>
```

Comment:
I saw with Roma, it's not the most convenient to not have it shown but at least the reply works for the moment.